src/pages/requesting-information.js

```diff
@@ -0,0 +1,287 @@
+import Helmet from "react-helmet";
+import GatsbyLink from "gatsby-link";
+import React from "react";
+
+import HeadingOne from "../components/HeadingOne";
+import HeadingTwo from "../components/HeadingTwo";
+import ListItem from "../components/ListItem";
+import MultilineTextControl from "../components/MultilineTextControl";
+import MultiSelectionControl from "../components/MultiSelectionControl";
+import OptionControl from "../components/OptionControl";
+import Paragraph from "../components/Paragraph";
+import SinglelineTextControl from "../components/SinglelineTextControl";
+import SingleSelectionControl from "../components/SingleSelectionControl";
+import Submit from "../components/Submit";
+import SubmitControls from "../components/SubmitControls";
+import Title from "../components/Title";
+import UnorderedList from "../components/UnorderedList";
+
+import "./index.css";
+
+const Pages = () => (
+  <div>
+    <Helmet>
+      <title>{`Requesting information`}</title>
+    </Helmet>
+    <GatsbyLink className="Pages-home" to="/">{`Home`}</GatsbyLink>
+    <div className="Pages-title">
+      <Title>{`Requesting information`}</Title>
+    </div>
+    <div className="Pages-paragraph">
+      <Paragraph
+      >{`When requesting information from a user you should:`}</Paragraph>
+    </div>
+    <div className="Pages-unorderedList">
+      <UnorderedList>
+        <ListItem>{`only ask for information you absolutely need`}</ListItem>
+        <ListItem
+        >{`mark the label with '(optional)' if you do ask for extra information`}</ListItem>
+        <ListItem
+        >{`avoid unnecessary words like 'Please' or 'Enter' in labels`}</ListItem>
+      </UnorderedList>
+    </div>
+    <div className="Pages-headingOne">
+      <HeadingOne>{`Text entry`}</HeadingOne>
+    </div>
+    <div className="Pages-paragraph">
+      <Paragraph
+      >{`Use the components below for text entry. You should:`}</Paragraph>
+    </div>
+    <div className="Pages-unorderedList">
+      <UnorderedList>
+        <ListItem>
+          {`make the size appropriate to the amount of information requested`}
+        </ListItem>
+        <ListItem>{`use multiples of 8 for setting the size`}</ListItem>
+        <ListItem>{`use hint text rather than placeholder text`}</ListItem>
+      </UnorderedList>
+    </div>
+    <div className="Pages-headingTwo">
+      <HeadingTwo>{`SinglelineTextControl`}</HeadingTwo>
+    </div>
+    <div className="Pages-example">
+      <div className="Pages-exampleItem">
+        <SinglelineTextControl
+          hint="As it appears on your ID."
+          id="text-control-full-name-example"
+          label="Full name"
+          size="48"
+        />
+      </div>
+    </div>
+    <div className="Pages-example">
+      <div className="Pages-exampleItem">
+        <SinglelineTextControl
+          id="text-control-post-code-example"
+          label="Postcode (optional)"
+          size="8"
```

Comment:
The example for the postcode is does not fit a capitalised postcode. ![image](https://user-images.githubusercontent.com/893208/37911535-38d040b6-3108-11e8-9e3a-6ebc8b1b54b2.png)